website/addons/wiki/routes.py

```diff
@@ -120,6 +120,23 @@
             '/project/<pid>/node/<nid>/wiki/<wname>/',
         ], 'delete', views.project_wiki_delete, json_renderer),
 
+        # Change Wiki Settings | PUT
+        Rule([
+            '/project/<pid>/wiki/settings/',
```

Comment:
Settings is better as a url from a rest standpoint because it denotes a collection. But if the other addons' api/v1 routes use config, then we should keep it consistent. V2 would use settings in this context, though.